sonyapilib/device.py

```diff
@@ -320,6 +355,18 @@ def _parse_ircc(self):
 
             self._ircc_categories.add(category_info.text)
 
+    @staticmethod
+    def _find_device_info(text, info, upnp_device=None):
+        upnp_device = upnp_device or f"{URN_UPNP_DEVICE}device"
```

Comment:
can you add a test where upnp_device is None to test if the search is working then?

Reply:
Will take a look at that in approximately an hour

Reply:
Done